### PR TITLE
Add information about HATEOAS alternatives

### PR DESCRIPTION
**Pull request by asbjornu**
The following note is false:

>Currently there are no standards or specifications that define how to model the HATEOAS principle.

[The Atom Publishing Protocol](https://tools.ietf.org/html/rfc5023) exists and has for some time. Although it's not general-purpose, it works well for CRUD-style APIs. I also think it's worth mentioning all the specifications that do exist, although none of them are ratified standards (yet).

### CODE REVIEW

**Comment by PRMerger18** [comment]
@asbjornu : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by ktoliver** [comment]
@dragon119 - Hi, would you review the proposed update to your article? Thanks.

File: docs/best-practices/api-design.md [hunk 1]
```diff
@@ -328,7 +328,7 @@ A subsequent request from the client application can retrieve the remainder of t
...
 
 > [!NOTE]
-> Currently there are no standards or specifications that define how to model the HATEOAS principle. The examples shown in this section illustrate one possible solution.
+> Currently there are no general-purpose standards that define how to model the HATEOAS principle. While [The Atom Publishing Protocol](https://tools.ietf.org/html/rfc5023) is an established standard for providing hypermedia controls, it is (as the name suggests) primarily designed for content publishing and thus best fit for [CRUD](https://en.wikipedia.org/wiki/Create,_read,_update_and_delete)-style APIs. While more general-purpose alternatives such as [HAL](http://stateless.co/hal_specification.html), [Siren](https://github.com/kevinswiber/siren), [Verbose](https://verbose.readthedocs.io/en/latest/index.html), [Mason](https://github.com/JornWildt/Mason), [Hyper](http://hyperjson.io/) and [Hydra](https://www.hydra-cg.com/) exist, none are ratified standards yet. The examples shown in this section illustrate one possible, proprietary solution.
```
Comment (ckittel) [inline docs/best-practices/api-design.md:331]: Any chance we can simplify to just?  We don't probably need to link up every hypermedia implementation out there.

```suggestion
> Currently there are no general-purpose standards that define how to model the HATEOAS principle.  The examples shown in this section illustrate one possible, proprietary solution.
```

**Comment by PRMerger9** [comment]
@doodlemania2 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by doodlemania2** [comment]
#sign-off

**Comment by PRMerger9** [comment]
@doodlemania2: I'm sorry - only the author of this article, @dragon119, can sign off on your changes. But we do have an exception process - if you are on the Microsoft content or product team for this product area, you can ask the PR review team to review and merge it by sending mail to the techdocprs alias.

**Pull request merged**